code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -287,23 +295,8 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 	var/mob/living/L = input(invoker, "Choose a fellow servant to heal.", name) as null|anything in nearby_cultists
 	if(!L || !invoker || !invoker.canUseTopic(slab))
 		return 0
-	var/brutedamage = L.getBruteLoss()
-	var/burndamage = L.getFireLoss()
-	var/totaldamage = brutedamage + burndamage
-	if(!totaldamage)
-		invoker << "<span class='warning'>[L] is not burned or bruised!</span>"
-		return 0
-	L.adjustToxLoss(totaldamage * 0.5)
-	L.adjustBruteLoss(-brutedamage)
-	L.adjustFireLoss(-burndamage)
-	var/healseverity = max(round(totaldamage*0.05, 1), 1) //shows the general severity of the damage you just healed, 1 glow per 20
-	var/targetturf = get_turf(L)
-	for(var/i in 1 to healseverity)
-		PoolOrNew(/obj/effect/overlay/temp/heal, list(targetturf, "#1E8CE1"))
 	invoker << "<span class='brass'>You bathe [L] in Inath-neq's power!</span>"
-	L.visible_message("<span class='warning'>A blue light washes over [L], mending [L.p_their()] bruises and burns!</span>", \
-	"<span class='heavy_brass'>You feel Inath-neq's power healing your wounds, but a deep nausea overcomes you!</span>")
-	playsound(targetturf, 'sound/magic/Staff_Healing.ogg', 50, 1)
+	L.apply_status_effect(STATUS_EFFECT_SENTINELS_BOON)
```

Comment:
That's partially the intention. Like a lot of Ratvar's scripture, this was very situational but beat every other option in that situation. I felt that a HoT was, although weaker overall, more useful in more situations; for instance, you could pre-HoT before a fight, get someone out of critical condition, or just recover from being shocked.